### PR TITLE
PES-2376 PHPstan level 4

### PR DESCRIPTION
**Pull request by Karpec**

### CODE REVIEW

File: src/Packetery/Module/WpdbAdapter.php [hunk 1]
```diff
@@ -325,7 +325,7 @@ public function getLastWpdbError(): string {
 	 * @return \Generator
 	 */
 	public function getWpdbQueries(): \Generator {
-		if ( isset( $this->wpdb->queries ) ) {
+		if ( count( $this->wpdb->queries ) > 0 ) {
```
Comment (jan-potuznik-z) [inline src/Packetery/Module/WpdbAdapter.php:328]: `TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /app/wcbeta/wp-content/plugins/packeta/src/Packetery/Module/WpdbAdapter.php:328`

File: src/Packetery/Module/Checkout.php [hunk 2]
```diff
@@ -512,7 +512,7 @@ public function validateCheckoutData(): void {
 				wc_add_notice( __( 'The selected Packeta carrier is not available for the selected delivery country.', 'packeta' ), 'error' );
 				$error = true;
 			}
-
+			// @phpstan-ignore-next-line
 			if ( ! $error && PickupPointValidator::IS_ACTIVE ) {
```
Comment (jan-potuznik-z) [inline src/Packetery/Module/Checkout.php:516]: Co lepší porovnání pro levou stranu?

File: src/Packetery/Module/WpdbAdapter.php (hunk 1 above)
Comment (Karpec) [reply src/Packetery/Module/WpdbAdapter.php]: Dobrý catch

File: src/Packetery/Module/Checkout.php (hunk 2 above)
Comment (Karpec) [reply src/Packetery/Module/Checkout.php]: Může být, když už jdu okolo :)

File: src/Packetery/Module/WpdbAdapter.php [hunk 3]
```diff
@@ -325,7 +325,8 @@ public function getLastWpdbError(): string {
 	 * @return \Generator
 	 */
 	public function getWpdbQueries(): \Generator {
-		if ( isset( $this->wpdb->queries ) ) {
+		if ( ( defined( 'SAVEQUERIES' ) && constant( 'SAVEQUERIES' ) === true )
```
Comment (jan-potuznik-z) [inline src/Packetery/Module/WpdbAdapter.php:328]: _Expressions in parentheses MAY be split across multiple lines, where each subsequent line is indented at least once. When doing so, the first condition MUST be on the next line. The closing parenthesis and opening brace MUST be placed together on their own line with one space between them. Boolean operators between conditions MUST always be at the beginning or at the end of the line, not a mix of both._ My používáme operátory na konci řádek.

**Comment by Karpec** [comment]
Uzavírám, mergneme level 2 strict rules až level 5 najednou do after-freez větve

**Pull request closed without merging**